src/components/layout/AppNavDrawer.vue

```diff
@@ -61,6 +61,14 @@
             {{ $t('app.general.title.history') }}
           </app-nav-item>
 
+          <app-nav-item
+            v-if="supportsTimelapse"
+            icon="$video"
+            to="/timelapse"
+          >
+            {{ $t('app.timelapse.title.timelapse') }}
```

Comment:
For consistency purposes, this should be a new key called `app.general.title.timelapse`